pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -568,13 +568,20 @@ func (k *Bootstrapper) needsReconfigure(conf string, hostname string, port int,
 		klog.Infof("needs reconfigure: configs differ:\n%s", rr.Output())
 		return true
 	}
-
-	st, err := kverify.APIServerStatus(k.c, hostname, port)
+	// cruntime.Enable() may restart kube-apiserver but does not wait for it to return back
```

Comment:
could we add this additional check into cruntime.Enable() instead ?

Reply:
alternatively consider adding a new helper in kverify

kverify.APIServerStatusWithRetry(... 

so the logic here be simple and short